Content.Server/_Goobstation/Devil/CheatDeath/CheatDeathSystem.cs

```diff
@@ -0,0 +1,145 @@
+// SPDX-FileCopyrightText: 2025 GoobBot <uristmchands@proton.me>
+// SPDX-FileCopyrightText: 2025 Solstice <solsticeofthewinter@gmail.com>
+// SPDX-FileCopyrightText: 2025 SolsticeOfTheWinter <solsticeofthewinter@gmail.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+using Content.Shared._Goobstation.DelayedDeath;
+using Content.Shared._Goobstation.CheatDeath;
+using Content.Shared._Goobstation.Devour.Events;
+using Content.Server._Shitmed.DelayedDeath;
+using Content.Server.Actions;
+using Content.Server.Administration.Systems;
+using Content.Server.Jittering;
+using Content.Shared.Damage;
+using Content.Shared.Examine;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Mobs.Components;
+using Content.Shared.Mobs.Systems;
+using Content.Shared.Popups;
+using Content.Shared.Traits.Assorted;
+using Robust.Shared.Network;
+
+namespace Content.Server._Goobstation.Devil.CheatDeath;
```

Comment:
Unused using directives
```suggestion
using Content.Shared._Goobstation.DelayedDeath;
using Content.Shared._Goobstation.CheatDeath;
using Content.Shared._Goobstation.Devour.Events;
using Content.Server.Actions;
using Content.Server.Administration.Systems;
using Content.Server.Jittering;
using Content.Shared.Damage;
using Content.Shared.Examine;
using Content.Shared.Mobs.Systems;
using Content.Shared.Popups;
using Content.Shared.Traits.Assorted;

namespace Content.Server._Goobstation.Devil.CheatDeath;
```